api/public_api_v1.yaml

```diff
@@ -237,33 +246,45 @@ definitions:
         description: Number of items returned for each page.
       next:
         type: string
-        description: 'Contains an URL to GET the next #<page_size> results in the retrieved collection of items.'
+        description: 'Contains an URL to GET the next #<page_size> results in the
+          retrieved collection of items.'
         format: uri
   LimitedProfile:
     title: A citizen's profile
-    description: Describes the citizen's profile, mostly interesting for preferences attributes.
+    description: Describes the citizen's profile, mostly interesting for preferences
+      attributes.
     type: object
     properties:
       preferred_languages:
-        description: 'Indicates the User''s preferred written or spoken language. Generally used for selecting a localized User interface. Valid values are concatenation of the ISO 639-1 two letter language code, an underscore, and the ISO 3166-1 2 letter country code; e.g., ''en_US'' specifies the language English and country US.'
-        $ref: "#/definitions/PreferredLanguages"
+        description: Indicates the User's preferred written or spoken language. Generally
```

Comment:
can the description be refactored into `#/definitions/PreferredLanguages` to avoid duplication?

Reply:
Also since this is an array, the description should read `Indicates the User's preferred written or spoken languages in order of preference.`